src/kili/use_cases/issue/__init__.py

```diff
@@ -0,0 +1,34 @@
+"""Issue Service."""
+
+from typing import List
+
+from kili.entrypoints.mutations.issue.helpers import get_labels_asset_ids_map
+from kili.gateways.kili_api_gateway import KiliAPIGateway
+from kili.gateways.kili_api_gateway.issue.types import IssueToCreateKiliAPIGatewayInput
+from kili.use_cases.issue.types import IssueToCreateUseCaseInput
+
+
+class IssueUseCases:
+    """Issue Service."""
+
+    def __init__(self, kili_api_gateway: KiliAPIGateway):
+        self._kili_api_gateway = kili_api_gateway
+
+    def create_issues(self, project_id, issues: List[IssueToCreateUseCaseInput]):
+        """Create issues with issue type."""
+        label_id_array = [issue.label_id for issue in issues]
+        label_asset_ids_map = get_labels_asset_ids_map(
+            self._kili_api_gateway, project_id, label_id_array
+        )  # should be done in the backend
+        gateway_issues = [
+            IssueToCreateKiliAPIGatewayInput(
+                issue_number=0,
```

Comment:
always 0?

Reply:
Since is it debt, it may be better to put that in the Gateway, no need to pollute the use case code.